.gitignore

```diff
@@ -20,3 +20,22 @@ integration/reports/*
 dashboard/src/locales/
 dashboard/public/clr-ui-dark.min.css*
 dashboard/public/clr-ui.min.css*
+cmd/kubeapps-apis/plugins/fluxv2/packages/v1alpha1/testdata/charts/podinfo-6.1.0.tgz
+cmd/kubeapps-apis/plugins/fluxv2/packages/v1alpha1/testdata/charts/podinfo-6.1.1.tgz
+cmd/kubeapps-apis/plugins/fluxv2/packages/v1alpha1/testdata/charts/podinfo-6.1.2.tgz
+cmd/kubeapps-apis/plugins/fluxv2/packages/v1alpha1/testdata/charts/podinfo-6.1.3.tgz
+cmd/kubeapps-apis/plugins/fluxv2/packages/v1alpha1/testdata/charts/podinfo-6.1.4.tgz
+cmd/kubeapps-apis/plugins/fluxv2/packages/v1alpha1/testdata/charts/podinfo-6.1.7.tgz
+cmd/kubeapps-apis/plugins/fluxv2/packages/v1alpha1/testdata/charts/podinfo-6.1.8.tgz
+cmd/kubeapps-apis/plugins/fluxv2/packages/v1alpha1/podinfo-6.1.5.tgz
+podinfo-6.1.5.tgz
+redocluster.sh
+cmd/kubeapps-apis/plugins/fluxv2/packages/v1alpha1/gist.md
+cmd/kubeapps-apis/plugins/fluxv2/packages/v1alpha1/mytest.sh
+foo.txt
+logs.sh
+mkall.sh
+mkapis.sh
+mkcontr.sh
+mksync.sh
+mytest.sh
```

Comment:
Is this added on purpose?

Reply:
it was, but lets not waste time on this. Dropped the file from PR list.